### DISCUSSION TITLE
how t uninstall podman-desktop dmg on mac/osx?

### DISCUSSION THREAD

**Discussion by tyoc213** [podman-desktop/podman-desktop#8644] [Q&A] [answered]
I installed podman from the dmg version.

If I installed core and compose from within the app removing `/Applications/Podman Desktop.app`? will remove those 2 components too?

**Comment by jeffmaury** [comment]
Don't know what is the core but the compose binary is installed under $HOME/.local/share/containers/podman-desktop/extension-storage/podman-desktop.compose

**Reply by tyoc213** [reply]
@jeffmaury this is installed under indicated `containers` folder

```
➜  containers tree podman
podman
└── machine
    ├── applehv
    │   └── cache
    └── libkrun
        └── cache

6 directories, 0 files
➜  containers tree podman-desktop 
podman-desktop
├── configuration
│   └── settings.json
├── contributions
├── extensions-storage
│   ├── podman-desktop.compose
│   │   └── bin
│   │       └── docker-compose
│   └── podman-desktop.podman
│       └── podman-ext.json
├── plugins
├── plugins-scanning
└── safe-storage
    └── data.json

10 directories, 4 files
```

**Reply by jeffmaury** [reply, marked as answer]
So if you want to remove Podman Desktop the podman-desktop folder should be removed as well
